status/flowcells.py

```diff
@@ -113,20 +115,39 @@ class FlowcellsInfoDataHandler(SafeHandler):
     """
     def get(self, flowcell):
         self.set_header("Content-type", "application/json")
-        self.write(json.dumps(self.flowcell_info(flowcell)))
+        flowcell_info = self.__class__.get_flowcell_info(self.application, flowcell)
+        self.write(json.dumps(flowcell_info))
 
-    def flowcell_info(self, flowcell):
-        fc_view = self.application.flowcells_db.view("info/summary2",
+    @classmethod
```

Comment:
Maybe we could use staticmethod instead?

Reply:
Fixed now!